### PR TITLE
fix: add display:none for hidden

### PR DESCRIPTION
**Pull request by davidoff31**
Add display: none rule to all UI components that apply a :host display rule as suggested https://developers.google.com/web/fundamentals/web-components/best-practices#add-a-host-display-style-that-respects-the-hidden-attribute

### CODE REVIEW

File: packages/input-datepicker/test/lion-input-datepicker.test.js [hunk 1]
```diff
@@ -493,6 +493,9 @@ describe('<lion-input-datepicker>', () => {
         expect(myOverlayOpenedCbHandled).to.be.true;
         await myElObj.selectMonthDay(1);
         expect(myUserSelectedChangedCbHandled).to.be.true;
+        /*         myElObj.overlayEl.hidden = true;
```
Comment (jorenbroekema) [inline packages/input-datepicker/test/lion-input-datepicker.test.js:496]: I think you forgot to clean up this comment? :P

File: packages/form-system/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+
```
Comment (jorenbroekema) [inline packages/form-system/CHANGELOG.md:6]: might be accidental change?

File: packages/input-datepicker/test/lion-input-datepicker.test.js (hunk 1 above)
Comment (davidoff31) [reply packages/input-datepicker/test/lion-input-datepicker.test.js]: Yes, I did! Made new commit to resolve.

File: packages/form-system/CHANGELOG.md (hunk 2 above)
Comment (davidoff31) [reply packages/form-system/CHANGELOG.md]: Definitely! Removed in the last commit. Thanks!

**Comment by jorenbroekema** [comment]
Thanks for the contribution! I've merged it to dev for now because this will release almost all of our packages, so I hope to bundle some other things with it to reduce the number of releases. I'll send an update once it's released so we can make sure it's fixed on the latest version of our extension layer.

**Pull request merged**